desktop/src/app.rs

```diff
@@ -385,7 +391,7 @@ impl App {
                         self.window.scale_factor(),
                     );
 
-                    self.window.set_inner_size(window_size);
+                    let _ = self.window.request_inner_size(window_size);
```

Comment:
I feel like caring about the return value here _could_ do something about #8870, but maybe not, and also, that can be done later. For now, I guess it doesn't matter, I'm not seeing any weird scaling behavior.

Reply:
I'd say let's keep current behaviour and consider coming back to try and fix that issue separately